src/search/ScrollTrackMarkers.js

```diff
@@ -188,6 +220,20 @@ define(function (require, exports, module) {
         }
     }
 
+    /**
+     * @return {number} amount of vertical space above and below the scrollbar, in pixels
+     */
+    function getScrollbarTrackOffset() {
+        return scrollbarTrackOffset;
+    }
+
+    /**
+     * @param {number} offset Amount of vertical space above and below the scrollbar, in pixels
+     */
+    function setScrollbarTrackOffset(offset) {
```

Comment:
I _only_ need the setter.
But if you (or anyone else) feel like these shouldn't be there, I will gladly remove them.